app/assets/src/components/utils/links.js

```diff
@@ -0,0 +1,16 @@
+const openUrl = link => {
+  let openInNewTab = false;
+
+  // metakey is CMD in mac
+  if (event && (event.metaKey || event.ctrlKey)) {
```

Comment:
This method doesn't have `event` parameter any more.
It might be good for the function to receive `openInNewTab` as a parameter directly.